Bensch97/Capstone-Reddit
# Style/voting

# Description
User can upvote/downvote posts on the following pages:
 - front page
- subreddit
- individual post
- user profile

Voting post requests are handled by AJAX so that page is not reloaded when voting. JQuery is used to change the color of voting buttons and to update vote score on post.

# Testing
Go to each view listed above and test upvote/downvote on posts. 
- Confirm the vote score is as expected.
- Confirm the button/arrow you click on turns orange (and only that button).
- Confirm page is not redirecting or reloading.
- Refresh each page after voting and confirm the score and button color remain the same.

# Extra
 Explore view was converted to a class-based view. We still need to do one more per Joe's request.
